modules/asr/Cyberon/FlashReadData.c

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021-2022 XMOS LIMITED.
```

Comment:
Date and newline

Reply:
Most of the files in the repo have the wrong date, I will update them after this PR is merged.